### ISSUE TITLE
Tabbing out of Games (Valorant especially)

### ISSUE THREAD

**Issue by DyyP** [ciderapp/Cider-2#452]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://cider.sh/legal/conduct) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ciderapp/Sabiiro-BugTracker/issues) for a bug report that matches the one I want to file, without success.
- [X] I have read the [troublshooting FAQs](https://cider.gitbook.io/welcome-to-gitbook/support/faqs) and none solved my issue.

### What operating system are you using?

Windows

### Build Details

2.12.6

### Operating System Version

Win 10

### Describe the Bug

When playing Valorant (fullscreen) with cider open it keeps tabbing me out of the game.
there is nothing happening in cider, sometimes music isnt even playing... 
it does that every 10min to 1h, seems pretty ramdom but most often in an ~15min intervall

### Steps to Reproduce

Play Val 

have Cider open, playing music or not 

so basically have fun gaming and jamming to your fav tunes for an hour XD

have fun

**Comment by DyyP** [comment]
whos your fav agend/who did you play :)

**Comment by coredev-uk** [comment]
Very weird bug, can't say that I've seen anyone else have this issue. Is there any certain page that you're on, or any main setting changes that you've made?

**Comment by coredev-uk** [comment]
This issue has been closed to consolidate the issue tracker for Cider 2.3.1 and later issues. If you believe this issue is still relevant, please open a new issue.
